cmd/limactl/inspect.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/goccy/go-yaml"
+	"github.com/lima-vm/lima/pkg/limayaml"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newInspectCommand() *cobra.Command {
+	inspectCommand := &cobra.Command{
+		Use:               "inspect [flags] [INSTANCE]...",
+		Short:             "Inspect instances of Lima.",
+		Args:              cobra.ArbitraryArgs,
+		RunE:              inspectAction,
+		ValidArgsFunction: inspectBashComplete,
+	}
+
+	inspectCommand.Flags().String("format", "yaml", "output format, one of: json, yaml")
```

Comment:
I'd prefer the default to be json for consistency with other commands (`limactl info`)

Reply:
No problem, I will change it.

Reply:
Changed to json